simpledotorg/simple-server
# Introduce Overdue Patients Returned Card on Dashboard

**Story card:** [sc-10784](https://app.shortcut.com/simpledotorg/story/10784/integrate-data-with-return-to-care-graph)

## Because

We want to see the impact of the calls made to overdue patients by measuring the number of patients who return to care after receiving the call.

## This addresses

This card introduces a new graph, which shows the percentage of overdue patients who received a call and returned to care. 

Additionally, it also shows a breakdown of the returned to care number by the call result (ie: 'agreed to visit', 'remind to call later' or 'removed from overdue list').

## Test instructions

Enable the feature flag (`overdue_patients_returned`) for your user or a user group to see this graph on this HTN dashboard.
